packages/core/stories/spinner/spinner.doc.stories.mdx

```diff
@@ -24,20 +24,20 @@ Improve accessibility by customizing the `aria-label` to provide additional cont
 
 ## Default
 
-Spinners have two sizes, `default` and `large`.
+Spinners have two sizes, `default` and `nested`.
 
 <Canvas>
   <Story id="core-spinner--default" />
 </Canvas>
 
-## Large
+## Nested
 
-Set the `size` prop to `"large"` to use the large Spinner.
+Set the `size` prop to `"nested"` to use the nested Spinner.
 
 Large Spinners can be beneficial for full screen loading experiences; while default Spinners can be reserved for widget loading experiences.
 
 <Canvas>
-  <Story id="core-spinner--large" />
+  <Story id="core-spinner--nested" />
```

Comment:
Say it with me "99% of the time, if the example has to change to accommodate a code change then it's a breaking change"

P.S. 67% of statistics are mad up on the spot 😉